packages/headless/src/features/commerce/pagination/pagination-slice.test.ts

```diff
@@ -91,6 +96,12 @@ describe('pagination slice', () => {
     expect(finalState.page).toBe(0);
   });
 
+  it('#setPageSize sets the page size', () => {
+    const finalState = paginationReducer(state, setPageSize(17));
+
+    expect(finalState.perPage).toBe(17);
+  });
```

Comment:
This test reflects a discrepancy between the action and the state. The action sets the page size, but the state that backs this data is called `perPage`.

Should we switch the naming to `setPerPage`, as well as the controller method on the pagination controller? Should we leave things as-is? Other suggestions? @fbeaudoincoveo @BenKandelaars

Reply:
I'm personally in favor of renaming the action and controller method to match the property name in the state / API.

Reply:
(if you do, please update the it descriptions accordingly as well)